autoload/SpaceVim/mapping.vim

```diff
@@ -157,7 +156,12 @@ function! SpaceVim#mapping#close_current_buffer() abort
     echohl WarningMsg
     echon 'save changes to "' . bufname(bn) . '"?  Yes/No/Cancel'
     echohl None
-    let rs = nr2char(getchar())
+    if len(a:000) > 0
+      let rs = get(a:000, 0)
+    else
+      echon 'save changes to "' . bufname(bn) . '"?  Yes/No/Cancel'
```

Comment:
no need to add extra message.

Reply:
'save changes to "' . bufname(bn) . '"?  Yes/No/Cancel'
The message above should only appear when the arguments are not given.
So I have removed this line by the new commit. Please check it.
`    echon 'save changes to "' . bufname(bn) . '"?  Yes/No/Cancel'`